### ISSUE TITLE
no bin-related analysis using only one binning method

### ISSUE THREAD

**Issue by bresyd** [jtamames/SqueezeMeta#353]
Hi guys,

I have been running the latest squeezemeta version only using metabat for the binning (`--nomaxbin` since maxbin is sometimes very slow) and once the pipeline gets to step 16 (DAStool) it starts complaining that there are no DAStool results (which makes sense since only one binning method was used):
```
[8 days, 11 hours, 36 minutes, 30 seconds]: STEP16 -> DAS_TOOL MERGING: 16.dastool.pl
Can't open /scratch/btschits/sqm_1.4/sqm_so245_B_hifiasm/results/maxbin directory
ERROR in STEP16-> 16.dastool.pl
Can't open /scratch/btschits/sqm_1.4/sqm_so245_B_hifiasm/results/DAS/sqm_so245_B_hifiasm_DASTool_bins directory, no DAStool results
WARNING: File /scratch/btschits/sqm_1.4/sqm_so245_B_hifiasm/results/DAS/sqm_so245_B_hifiasm_DASTool_bins/ is empty!. DAStool did not generate results
Skipping BIN TAX ASSIGNMENT: DAS_Tool did not predict bins.
Skipping CHECKM: DAS_Tool did not predict bins.
Skipping BIN TABLE CREATION: (You already know: DAS_Tool did not predict bins.)
```
However, I thought that SQM would still do all the binning-related analysis (taxonomy, checkm, creating the table) but only on the metabat instead of the DAStool output when using the '--nomaxbin' flag? Or is what I am observing the default behavior of SQM (in which case I am wondering if that can be changed)?

Thanks a lot for your help.

**Comment by jtamames** [comment]
Hello!
No, the intended behavior is to go on without DAStool merging. Could I take a look at your `syslog` and `SqueezeMeta_conf.pl `files?
Best,
J

**Comment by bresyd** [comment]
[SqueezeMeta_conf.pl.gz](https://github.com/jtamames/SqueezeMeta/files/7082340/SqueezeMeta_conf.pl.gz)
[syslog.gz](https://github.com/jtamames/SqueezeMeta/files/7082342/syslog.gz)


Hi Javier,

thanks for your quick reply. Please find the files attached.

Cheers

**Comment by jtamames** [comment]
Hello
That looks like a pretty silly mistake from our side. For making this one able to run, it will be enough to change line 27 in your SqueezeMeta_conf.pl, replacing it by:

`%bindirs = ("metabat2","$resultpath/metabat2");`

I will fix this bug in the code asap. Thanks for pointing it out!
Best,
J

**Comment by bresyd** [comment]
Hi Javier,



thanks for these instructions. Looks good so far, checkM is currently running. I will let you know if everything worked once the run is finished.




cheers



-----Original message-----

**Comment by bresyd** [comment]
Hi Javier,
all good now, the remaining steps were performed as expected. 
Thanks again for your help

**Comment by jtamames** [comment]
Nice to hear that. This bug will be fixed in the upcoming 1.5 version.
Best,
Javier
